internal/pkg/ecs/ecs.go

```diff
@@ -232,6 +247,50 @@ func (c Client) StopDefaultClusterTasks(familyName string) error {
 	return c.ecsClient.StopTasks(taskIDs, ecs.WithStopTaskReason(taskStopReason))
 }
 
+// DecryptedSSMSecrets returns the decrypted parameters from the SSM store.
+func (c Client) DecryptedSSMSecrets(secrets []*ecs.ContainerSecret) ([]EnvVar, error) {
```

Comment:
Can we combine this one with `DecryptedSecretManagerSecrets` since

- these two take the same input and output type
- we almost always call `DecryptedSSMSecrets` and `DecryptedSecretManagerSecrets` together

so that we don't need to iterate twice. The output can be a struct with decrypted SSM secrets and secret manager secrets.